Library/Homebrew/formula_installer.rb

```diff
@@ -407,7 +407,8 @@ def install_dependencies(deps)
     if deps.empty? && only_deps?
       puts "All dependencies for #{formula.full_name} are satisfied."
     else
-      oh1 "Installing dependencies for #{formula.full_name}: #{Tty.green}#{deps.map(&:first)*", "}#{Tty.reset}" unless deps.empty?
+      oh1 "Installing dependencies for #{formula.full_name}"
```

Comment:
Would it make sense to address this instead by allowing `oh1` to accept options and to optionally disable the truncation, i.e. in cases like these where we want the styling but not the truncation even in non-verbose mode? E.g. something like this:

```ruby
def oh1(title, options = {})
  if $stdout.tty? && !ARGV.verbose? && options.fetch(:allow_truncation, true)
    title = Tty.truncate(title)
  end
  puts "#{Tty.green}==>#{Tty.white} #{title}#{Tty.reset}"
end

oh1 "I shall not be truncated", :allow_truncation => false
```